lib/MusicBrainz/Server/Controller/Root.pm

```diff
@@ -71,6 +71,9 @@ sub set_language : Path('set-language') Args(1)
     } else {
         # set the cookie to expire in a year
         $c->set_language_cookie($lang);
+        $c->flash->{message} =
```

Comment:
This functionality is provided by http://search.cpan.org/~jjnapiork/Catalyst-Plugin-Session-0.40/lib/Catalyst/Plugin/Session.pm